packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "17.0.1",
+  "version": "17.1.0",
```

Comment:
These updates are probably just a [patch](https://github.com/department-of-veterans-affairs/component-library#how-to-choose-a-version-number).